src/Misc/PerformanceViewer/performanceViewerCollectionStrategies.ts

```diff
@@ -201,10 +203,12 @@ export class PerfCollectionStrategy {
      * @returns the initializer for the absolute fps strategy
      */
     public static AbsoluteFpsStrategy(): PerfStrategyInitialization {
-        return () => {
+        return (scene) => {
+            const sceneInstrumentation = new SceneInstrumentation(scene);
+            sceneInstrumentation.captureFrameTime = true;
             return {
                 id: "absolute fps",
-                getData: defaultGetDataImpl,
+                getData: () => 1000.0 / sceneInstrumentation.frameTimeCounter.lastSecAverage,

```

Comment:
I use lastSecondAverage a lot, I wonder if it would be better to use current instead of lastSecAverage given that my use case is real time data, vs the statistics tab being "every 500ms" data. Any thoughts?

Reply:
yes, current value would be the right one for a realtime counter

Reply:
Done!